### DISCUSSION TITLE
How to export realm & users without default

### DISCUSSION THREAD

**Discussion by rgarrigue** [keycloak/keycloak#23775] [Q&A]
Hello

I'ld like to know how I can export realm & users settings without Keycloak's default. I'm aiming at a minimal configuration to get new keycloaks started for dev environments, and our current "legacy" setup export is 3000 lines long plus no one know the full picture to make one from scratch.

**Comment by schrieveslaach** [comment]
@rgarrigue, did you find an answer to this question?

**Comment by rgarrigue** [comment]
No. But I can close the issue, as we got something working for our use case. Nothing pretty, but a clean solution isn't worth it.

**Reply by schrieveslaach** [reply]
Thanks for the heads up. I was hoping to have a leaner Keycloak CLI config for our use cases.

**Reply by pantherwelt** [reply]
@rgarrigue Something like what ? I am planning to remove objects that have the attribute "description" given like "description": "${role_uma_authorization}", because I am assuming that those are defaults.
